File: components/redis-handling-session/src/main/java/com/sv/youapp/component/authorization/entities/redis/OAuth2NativeGrantAuthorization.java
package com.sv.youapp.component.authorization.entities.redis;

public class OAuth2NativeGrantAuthorization extends AbstractGrantAuthorization {

	public OAuth2NativeGrantAuthorization(
		String id,
		String registeredClientId,
		String principalName,
		String scopes,
		AbstractGrantAuthorization.AccessToken accessToken,
		AbstractGrantAuthorization.RefreshToken refreshToken
	) {
		super(id, registeredClientId, principalName, scopes, accessToken, refreshToken);
	}
}

File: components/redis-handling-session/src/main/kotlin/com/sv/youapp/component/authorization/entities/redis/OAuth2NativeGrantAuthorization.kt
package com.sv.youapp.component.authorization.entities.redis

class OAuth2NativeGrantAuthorization(
    id: String,
    registeredClientId: String,
    principalName: String,
    scopes: String,
    accessToken: AccessToken?,
    refreshToken: RefreshToken?,
) : AbstractGrantAuthorization(
        id,
        registeredClientId,
        principalName,
        scopes,
        accessToken,
        refreshToken,
    )
